### PR TITLE
[CdapIO] Complete examples for CDAP Hubspot plugins 

### PR DESCRIPTION
**Pull request by Amar3tto**
Added examples for the following CDAP Hubspot plugins:

- Batch Source
- Batch Sink
- Streaming Source

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Amar3tto** [comment]
Run Java_Clickhouse_IO_Direct PreCommit

**Comment by Amar3tto** [comment]
Run Java_Kafka_IO_Direct PreCommit

**Comment by Amar3tto** [comment]
Run Java_Pulsar_IO_Direct PreCommit

**Comment by Amar3tto** [comment]
Run Java PreCommit

File: examples/java/cdap/hubspot/src/main/java/org/apache/beam/examples/complete/cdap/utils/JsonElementCoder.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.apache.beam.sdk.coders.CustomCoder;
+import org.apache.beam.sdk.coders.StringUtf8Coder;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/** Custom {@link org.apache.beam.sdk.coders.Coder} for {@link JsonElement}. */
+public class JsonElementCoder extends CustomCoder<JsonElement> {
```
[the remark this answers is not in the document]
Comment (Amar3tto) [reply examples/java/cdap/hubspot/src/main/java/org/apache/beam/examples/complete/cdap/utils/JsonElementCoder.java]: Moved

**Comment by Amar3tto** [comment]
Run Java_Pulsar_IO_Direct PreCommit

**Comment by Lizzfox** [comment]
R: @johnjcasey
R: @chamikaramj 

We divided our https://github.com/apache/beam/pull/23501 by example (service). This one is for the Hubspot examples.
Could you please take the time to review it?

Thank you for your attention to it!

**Comment by Amar3tto** [comment]
Run Java_HCatalog_IO_Direct PreCommit

**Comment by Amar3tto** [comment]
Run Python PreCommit

**Comment by Amar3tto** [comment]
Run Java PreCommit

**Comment by Lizzfox** [comment]
@johnjcasey
We would really appreciate it if you would take the time to review the PR with the Hubspot example.
Thank you for your time and consideration!

**Review by johnjcasey — APPROVED** [review]

**Pull request merged**